### PR TITLE
fix: test suite

### PR DESCRIPTION
**Pull request by stephenmathieson**
This patch updates the build process and fixes the test suite.

Notable changes:

- The "grunt" dependency has been removed (along with `Gruntfile.js`); instead, we just call `browserify` directly.
- Babel is run via the `babelify` transform.
- The CSS (services.css) is now properly included in the build.
- The `webcomponents.js` dependency is now imported into the build rather than arbitrary copied into the `dist/` directory.

This is required before we can tackle #29.

/cc @WilcoFiers

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -21,12 +22,22 @@
   "devDependencies": {
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "babelify": "^8.0.0",
```
Comment (WilcoFiers) [inline example/package.json]: This was complaining that I needed babel-core. Do we need to add that?

Comment (stephenmathieson) [reply example/package.json]: Whoops, yes. It's a peer dependency of `babelify`. Will update.

**Comment by stephenmathieson** [comment]
> Can you update the commit message as well?

Are you able to adjust the commit message when squashing this in? I am still not super familiar with "conventional commits", but their documentation suggests that the "final burden" of commit messages [falls on the maintainer(s) of the project](https://conventionalcommits.org/#when-you-used-a-type-not-of-the-spec-eg-feet-instead-of-feat).

If not, I can adjust the commit message myself and force push, but this seems like an additional (and unnecessary) burden.

**Comment by WilcoFiers** [comment]
> Are you able to adjust the commit message when squashing this in?

Happy to squash. You can use `git commit --amend` to update your last commit, than force push. the update.

**Pull request merged**